docs/Wdio_Utility.md

```diff
@@ -62,76 +61,54 @@ There are a few things to note about the webdriver.io configuration provided by
-- `Terra.should.validateElement()` mocha-chai convenience method that takes a screenshot and verifies the images are within the specified mis-match tolerance and performs accessibility validation. Note: this method provides its own mocha it test case. Also, since viewports isn't accepted in this method, you need to [test in multiple viewports](#testing-multiple-viewports) This method accepts these arguments (in this order):
-    - String (optional): the test case name. Default name is 'default'
-    - Object (optional): the test options. Options include selector, misMatchTolerance and axeRules:
-         - selector: the element selector to take a screenshot of. Defaults to the global terra.selector.
-         - misMatchTolerance: number between 0 and 100 that defines the degree of mismatch to consider two images as identical, increasing this value will decrease test coverage. Defaults to the global visualRegression.compare.misMatchTolerance.
-         - axeRules: the axe rules to use as overrides if necessary.
-    - See [validateElement-spec.js](https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/validateElement-spec.js) for example usage.
-- `Terra.should.themeEachCustomProperty()` mocha-chai convenience method that runs a visual comparison test for each custom property given. Note: this method provides its own mocha it test case. The methods accepts these arguments (in this order):
-    - String (optional): the element selector to take a screenshot of. Defaults to the global terra.selector.
-    - Object: list of themeable-variable key-value pairs such that the key is the themeable-variable name and the value is the css value to check in the screenshot.
-  - `Terra.should.themeCombinationOfCustomProperties()` mocha-chai convenience method that runs a visual comparison test for a grouping of custom properties provided. Note: this method provides its own mocha it test case. The methods accepts these arguments (in this order):
-      - Object: the test options. Options include testName, selector and properties:
-           - testName (optional): the name associated to the test. Used to create unique screenshots. Defaults to `themed`.
-           - properties (required): object of themeable-variable key-value pairs such that the key is the themeable-variable name and the value is the css value to check in the screenshot.
-           - selector (optional): the element selector to take a screenshot of. Defaults to the global terra.selector.
+    - This helper can be used inside mocha `it` blocks by being called through `Terra.validates.screenshot()`. This takes the same arguments.
+- `Terra.hideInputCaret()` mocha-chai convenience method that sets the CSS caret color of an element to transparent. Normally this caret will blink when an editable text field is focused which can lead to inconsistent test failures. The method accepts this argument:
```

Comment:
We need to specify this must be placed within an Mocha before, beforeEach or it block

Reply:
Done in 04987d77e2d549eee0d8c03ad3f07fe0fa4103c2